pulp_smash/tests/platform/api_v2/simple_repo_test.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Test for basic repo creating functionality.
+"""
```

Comment:
If a docstring fits on one line, do this:

    """Ask a foo for a bar."""

If a docstring has a summary line and a body, do this:

    """Ask a foo for a bar.

    Ensure that the bar has a biz and a baz.

    """